packages/braid-design-system/project.json

```diff
@@ -20,17 +21,32 @@
         "script": "codemod"
       }
     },
+    "generate": {
+      "dependsOn": ["generate:icons", "generate:snippets"],
+      "executor": "nx:noop"
+    },
     "generate:icons": {
-      "inputs": ["{projectRoot}/icons/*.svg", "{projectRoot}/scripts"],
-      "outputs": ["{projectRoot}/src/**/icons/*/*Svg.tsx"],
+      "inputs": ["{projectRoot}/icons/*.svg", "scripts"],
+      "outputs": [
+        "{projectRoot}/src/**/components/icons/*/*Svg.tsx",
+        "{projectRoot}/src/**/components/icons/index.ts"
+      ],
       "executor": "nx:run-script",
       "options": {
         "script": "generate:icons"
       }
     },
+    "generate:snippets": {
+      "inputs": ["{projectRoot}/src/**/*.snippets.tsx", "scripts"],
+      "outputs": ["{projectRoot}/src/**/playroom/snippets.ts"],
```

Comment:
I think this should take the new snippets directory into account as an output, e.g. `"{projectRoot}/src/**/playroom/snippets/*.ts"`